notebooks/04_model_select_and_optimize/hypertune_spark_deep_dive.ipynb

```diff
@@ -531,39 +1033,39 @@
     "    space=space,\n",
     "    algo=tpe.suggest,\n",
     "    trials=trials,\n",
-    "    max_evals=10\n",
+    "    max_evals=25\n",
     ")\n",
     "                  \n",
     "time_hyperopt = time.time() - time_start"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 75,
+   "execution_count": 62,
    "metadata": {},
    "outputs": [
     {
      "data": {
```

Comment:
About this:
`For example, hyperopt avails Tree of Parzen Estimators (TPE) method for searching optimal parameters, which is more efficient than random search or grid search (details can be found here)`.
Maybe we should add a short explanation of what TPE does, or what are the benefits claimed?

Reply:
Good idea. Added.